src/Adapter/MSTest.CoreAdapter/Resources/xlf/Resource.cs.xlf

```diff
@@ -5,306 +5,306 @@
         <target state="translated">UTA031: Třída {0} neobsahuje platnou vlastnost TestContext. Vlastnost TestContext musí být typu TestContext, musí být nestatická, veřejná a nemůže být určena jenom pro čtení. Například: public TestContext TestContext.</target>
-        <note></note>
+        <note />
       </trans-unit>
       <trans-unit id="UTA_MethodDoesNotExists">
         <source>Method {0}.{1} does not exist.</source>
         <target state="translated">Metoda {0}.{1} neexistuje.</target>
-        <note></note>
+        <note />
       </trans-unit>
       <trans-unit id="TestAssembly_FileDoesNotExist">
         <source>File does not exist: {0}</source>
         <target state="translated">Neexistující soubor: {0}</target>
-        <note></note>
+        <note />
       </trans-unit>
       <trans-unit id="UTA_TypeLoadError">
         <source>Unable to get type {0}. Error: {1}.</source>
         <target state="translated">Nepodařilo se získat typ {0}. Chyba: {1}.</target>
-        <note></note>
+        <note />
       </trans-unit>
       <trans-unit id="TypeLoadFailed">
         <source>Unable to load types from the test source '{0}'. Some or all of the tests in this source may not be discovered.</source>
         <target state="translated">Nepodařilo se načíst typy ze zdroje testu {0}. Některé nebo všechny testy v tomto zdroji nemusí být zjištěné.</target>
-        <note></note>
+        <note />
       </trans-unit>
       <trans-unit id="UTA_ErrorMultiClassClean">
         <source>UTA026: {0}: Cannot define more than one method with the ClassCleanup attribute inside a class.</source>
         <target state="translated">UTA026: {0}: Uvnitř třídy nejde definovat více jak jednu metodu s atributem ClassCleanup.</target>
-        <note></note>
+        <note />
       </trans-unit>
       <trans-unit id="TestContextMessageBanner">
         <source>TestContext Messages:</source>
-        <target state="new">TestContext Messages:</target>
-        <note></note>
+        <target state="translated">Zprávy pro TestContext:</target>
+        <note />
       </trans-unit>
       <trans-unit id="UTA_CleanupMethodThrowsGeneralError">
         <source>Error calling Test Cleanup method for test class {0}: {1}</source>
-        <target state="new">Error calling Test Cleanup method for test class {0}: {1}</target>
-        <note></note>
+        <target state="translated">Při volání čisticí metody testu pro třídu {0} došlo k chybě: {1}</target>
+        <note />
       </trans-unit>
       <trans-unit id="UTA_CleanupStackTrace">
         <source>TestCleanup Stack Trace</source>
-        <target state="new">TestCleanup Stack Trace</target>
-        <note></note>
+        <target state="translated">Trasování zásobníku čištění testu</target>
+        <note />
       </trans-unit>
       <trans-unit id="DiscoveryWarning">
         <source>[MSTest][Discovery][{0}] {1}</source>
```

Comment:
@jayaranigarg Any idea why is this in resx if we don't expect it to get localized ?

Reply:
@v-zbsail : Can you check why this string is not translated, even though it is marked as translated?

Reply:
Do you refer to "TestCleanup Stack Trace"?

Reply:
@janbaltus : I meant "[MSTest][Discovery][{0}] {1}" string..

Reply:
Is this string to be translated?
Can you please add more context?

Reply:
Yes, I believe it should have been translated like the rest of the strings.
We plan to remove [MSTest] from this string though as we do not want MSTest to be translated but would want [Discovery] to be translated. However, the change of removing [MSTest] will be done in a separate PR.

Reply:
Ince you update the xlf we will fix this in next round of localization.
Please remove also Discovery or add note for that string explaining this is localizable.
Any string in [] {} look like placeholder from our point of view

Reply:
Thanks @janbaltus . Will do those changes later. Meanwhile I will push this PR in.